compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -198,7 +198,7 @@ enumerateMatmulTileArm64(TypeRange elementTypes, ExecutableTargetAttr target) {
   }
 
   // Fallback - no architecture-optimized tile size for this case.
-  return {};
+  return {TileMxNxK{8, 8, 4}};
```

Comment:
Revert that line - that was a debugging helper. `{}` is the normal fallback value to return here.

Reply:
removed.